test/integration/scheduled-action-integration.ts

```diff
@@ -325,5 +392,67 @@ describe('Scheduled Actions API', async function () {
 
       expect(updatedAction.scheduledFor.timezone).to.equal('Europe/Berlin')
     })
+
+    describe('for AggregateRoot annotated entry', () => {
```

Comment:
nit: could we have a test covering an entry that is not page, but is providing the payload?

Reply:
what would be the expected test case? Aren't we testing server in the sdk's repository?

Reply:
I agree, we already cover this scenario in the backend and this PR hasn't make any changes to the way errors are handled

Reply:
We should describe this in documentation and provide an example + error response structure and this should do